stetho/src/main/java/com/facebook/stetho/inspector/elements/android/ViewDescriptor.java

```diff
@@ -10,19 +10,132 @@
 package com.facebook.stetho.inspector.elements.android;
 
 import android.view.View;
+import android.view.ViewDebug;
+import com.facebook.stetho.common.LogUtil;
 import com.facebook.stetho.common.StringUtil;
 import com.facebook.stetho.common.android.ResourcesUtil;
-import com.facebook.stetho.inspector.elements.AttributeAccumulator;
 import com.facebook.stetho.inspector.elements.AbstractChainedDescriptor;
+import com.facebook.stetho.inspector.elements.AttributeAccumulator;
+import com.facebook.stetho.inspector.elements.StyleAccumulator;
 
 import javax.annotation.Nullable;
+import javax.annotation.concurrent.GuardedBy;
+
+import java.lang.annotation.Annotation;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
 import java.util.Map;
+import java.util.Set;
+import java.util.regex.Pattern;
 
 final class ViewDescriptor extends AbstractChainedDescriptor<View> implements HighlightableDescriptor {
   private static final String ID_ATTRIBUTE_NAME = "id";
-
+  private static final Pattern CAPITAL_PATTERN = Pattern.compile("(?=\\p{Upper})");
```

Comment:
We shouldn't have this as a `static` so that it can be GC'd once all peers disconnect from Stetho.

And, we should defer initialize it so that we don't pay for it until/unless the user clicks on the Styles stuff.

```
/**
  * NOTE: Only access this via {@link #getCapitalPattern}.
  */
private @Nullable Pattern mCapitalPattern;

private Pattern getCapitalPattern() {
  if (mCapitalPattern == null) {
    mCapitalPattern = Pattern.compile(...);
  }
  return mCapitalPattern;
}
```

(We don't really need to use double-checked-null and synchronized here because if 2 threads race to initialize this then we'll just end up with a wasted call to `Pattern.compile` and the object instance it returns. It won't affect correctness. It's unlikely, and not a big deal if it does happen, and we're probably better off saving the CPU cycles by not worrying about it.)